### DISCUSSION TITLE
Keyboard Colors

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1690] [Q&A]
How to change the Keyboard Back Ground Color?

**Comment by patrickgold** [comment]
Change the color of "Common background"-variable or of the background property in the "Keyboard window"-property (0.3.14)

**Reply by shary998** [reply]
where are those? I couldn't find

**Reply by patrickgold** [reply]
They are located in the theme editor, relatively at the top (assuming you use either the latest rc or 0.3.14 stable):

![image](https://user-images.githubusercontent.com/19412843/160014895-ac7ebc7f-517e-43cb-b825-756c21111bd0.png)

Or here:

![image](https://user-images.githubusercontent.com/19412843/160014956-19ec3e4e-1d35-4c49-87ed-338d5f6890a6.png)

**Comment by shary998** [comment]
No I'm talking about the code! From where I can change the default Keyboard background color? or change its search button color?

**Reply by patrickgold** [reply]
Changing pretty much every color of the keyboard UI is exactly what themes were made for, no need to change a thing in the source code. I don't really understand your goal I am sorry

**Reply by shary998** [reply]
I just want the default keyboard color to be blue. How can I do that?
Plus the latest build is showing an error regarding cMake file how to solve that?

**Reply by shary998** [reply]
![image](https://user-images.githubusercontent.com/79845597/160017559-81f636d3-2749-4d75-ad05-e80aa79ab15d.png)

**Reply by shary998** [reply]
As I am using cMake first time and have no idea about it kindly help!
